### PR TITLE
Migrate to standalone repository

### PR DESCRIPTION
**Pull request by elinscott**
Changes to support `utils` becoming a standalone repository. These incude...

- migrating the files to the base directory
- adding a `README.rst`
- adding support for `cmake` 
- adding a GitHub action to test `cmake`

### CODE REVIEW

**Comment by nscolonna** [comment]
Thanks, I'll have a look at this asap

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+| ``merge_evc.x`` - a program for merging wavefunction files
+| ``wann2kcp.x`` - a program for converting ``Wannier90`` files into a format readable by ``kcp.x``
+| ``epsilon.x`` - a modified version of ``Quantum ESPRESSO``'s ``epsilon.x``
+
+Installation
+------------
+
+First, ensure you have a local installation of ``Quantum ESPRESSO`` on your machine. See https://gitlab.com/QEF/q-e for how to do this.
+
+Having installed ``Quantum ESPRESSO``, ``koopmans-qe-utils`` can then be installed with ``cmake``:
```
Comment (nscolonna) [inline README.rst:17]: I guess the cmake requires QE to be compiled with Cmake, right? shall we say it explicitely here?

**Review by nscolonna — COMMENTED** [review]
Tested the compilation with ifort and it looks ok! 
regarding the Cmake see my comment on the README.rst

File: README.rst (hunk 1 above)
Comment (elinscott) [reply README.rst:17]: Done!

**Review by nscolonna — APPROVED** [review]
Thanks great job!

**Pull request merged**